anisoap/utils/moment_generator.py

```diff
@@ -129,8 +129,11 @@ def compute_moments_inefficient_implementation(A, a, maxdeg):
         <x^{n_0} * y^{n_1} * z^{n_2}> = \int(x^{n_0} * y^{n_1} * z^{n_2}) * \exp(-0.5*(r-a).T@cov@(r-a)) dxdydz
         \sum_{i=1}^{\\infty} x_{i}
 
-        Note that the term "moments" in probability theory are defined for normalized Gaussian distributions.
-        Here, we take the Gaussian
+        Note that the term "moments" in probability theory are defined for normalized Gaussian distributions. These
+        recursive relations find the moments of a normalized Gaussian, but we actually want to find the moments of
+        the unnormalized underlying gaussian (as seen in the equation above) because calculating expansion
```

Comment:
Does knowing moments of unnormalized Gaussian expansions help determine the coefficients? How?

Reply:
Yes, this is necessary. We do want to find the moments of the unnormalized gaussian, since we're taking the moments of an effective gaussian that isn't unnormalized.

Reply:
So our goal is to find the moments of unnormalized guassian from normalized ones? Just to clarify, we are finding unnormalized guassian moments because they are the actual representations coarse-grained system?

Reply:
The unnormalized gaussian moments aren't the actual representation, but is a major component.

Oversimplified, but if $c$ is our actual representation, and $c = a * b * ...$, the moments can be thought of as the "$a$" component.